ruma-identifiers-validation/src/error.rs

```diff
@@ -28,22 +31,19 @@ pub enum Error {
 
     /// The ID is missing the correct leading sigil.
     MissingLeadingSigil,
-
-    /// The key algorithm is not recognized.
-    UnknownKeyAlgorithm,
 }
 
 impl Display for Error {
     fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
         let message = match self {
             Error::EmptyRoomVersionId => "room version ID is empty",
             Error::InvalidCharacters => "localpart contains invalid characters",
+            Error::InvalidKeyAlgorithm => "unknown key algorithm specified",
```

Comment:
We're allowing unknown key algorithms, right? So this would just be for empty algorithms?

```suggestion
            Error::InvalidKeyAlgorithm => "invalid key algorithm specified",
```